src/js/Scroll.js

```diff
@@ -5,25 +5,46 @@ smoothscroll.polyfill();
 
 class Scroll {
   constructor() {
-    // value in pixels elements scrolled to will be from top of the window
-    this.scrollOffset = -60;
+    this.scrollOffset = 0;
 
-    $(document).ready(this.onReady.bind(this));
+    this.isClearHash = true;
+    this.clearHashTimeout = 0;
+
+    this.$header = $('#header');
 
+    $(document).ready(this.onReady.bind(this));
   }
 
   onReady() {
-    // fire hash change on load
-    this.onHashChange();
-    // and watch for hash changes
-    window.addEventListener('hashchange', this.onHashChange.bind(this), false);
+    // set scrollOffset
+    this.setScrollOffset();
+
+    // watch for scroll
+    window.addEventListener('scroll', this.onScroll.bind(this), false);
```

Comment:
This is what i'm talking about.

Reply:
Yh that was example code from another recent project. Could you show example of how you mean or commit the fix? I don't quite get what I need to change

Reply:
I just pushed a fix